src/AerospikeCache.php

```diff
@@ -77,15 +73,18 @@ protected function doClear($namespace = ''): bool
             $clearNamespace = function ($record) use ($namespace): void {
                 if ($namespace === mb_substr($record['key']['key'], 0, mb_strlen($namespace))) {
                     $status = $this->aerospike->remove($record['key']);
+
                     if (!$this->isStatusOkOrNotFound($status)) {
```

Comment:
Thats both an overkill and a bad exception usage.. 
If you just want to return `false` if any record is not cleared you might use reference out of the callback.

```php
function($record) use (&$status_code) {
    // ...
    if (!isOk()) {
       $status_code = false;
    }
}
```

Reply:
Well the idea was to stop the clear attempt if there is an unexpected error. I would say that is what exception is for. But ok changed it on a reference